mne/io/base.py

```diff
@@ -425,6 +425,30 @@ def _read_segment_file(self, data, idx, offset, fi, start, stop,
         raise NotImplementedError
 
     @verbose
+    @deprecated("This method has been renamed 'load_data' and will be removed "
+                "in v0.11.")
+    def preload_data(self, verbose=None):
+        """Preload raw data
+
+        Parameters
+        ----------
+        verbose : bool, str, int, or None
+            If not None, override default verbose level (see mne.verbose).
+
+        Returns
+        -------
+        raw : instance of Raw
+            The raw object with data.
+
+        Notes
+        -----
+        This function will load raw data if it was not already preloaded.
+        If data were already preloaded, it will do nothing.
+        """
+        if not self.preload:
+            self._preload_data(True)
+        return self.load_data()
```

Comment:
I meant that the entire contents of the function could just be `return self.load_data()`. Usually when deprecating if possible you should just entirely wrap to the new function. As it stands now you're duplicating executed checks here and in `load_data`. The way you had it before, the code was not as DRY. doing `def preload_data(self): return self.load_data()` doesn't have either problem.